src/sentry/conf/server.py

```diff
@@ -2762,3 +2762,5 @@ def build_cdc_postgres_init_db_volume(settings):
 
 SERVER_COMPONENT_MODE = os.environ.get("SENTRY_SERVER_COMPONENT_MODE", None)
 FAIL_ON_UNAVAILABLE_API_CALL = False
+
+DISALLOWED_CUSTOMER_DOMAINS = []
```

Comment:
It would be good to update the routes file that denies org based on slug with this list once it is populated so that we don't have two lists of accounts that are denied.

Reply:
@markstory I'm now dynamically generating the routes from this list in https://github.com/getsentry/getsentry/pull/8155/files